org.mamedev.MAME.yaml

```diff
@@ -0,0 +1,61 @@
+id: org.mamedev.MAME
+runtime: org.kde.Platform
+runtime-version: 5.15-21.08
+sdk: org.kde.Sdk
+command: mame
+separate-locales: false
+finish-args:
+  - --device=all
+  - --persist=.mame
+  - --filesystem=home
+  - --filesystem=host:ro
+  - --share=network
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+  - --socket=wayland
+  - --allow=bluetooth
+modules:
+  - name: mame
+    buildsystem: simple
+    build-options:
+      no-debuginfo: true
+      strip: true
+    build-commands:
+      - make #-j ${FLATPAK_BUILDER_N_JOBS}
```

Comment:
Was it broken when multithreaded?

Reply:
Yes, build failed.  The MAME build system was just recently reworked, I am planning to try to re-enable when the next version releases in the next few days